### PR TITLE
added Grafana docker

### LINKED ISSUE #11: Create a Docker for Grafana
Create a Docker for Grafana

### PR DESCRIPTION
**Pull request by Avev**
resolved #11

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -6,4 +6,16 @@ services:
       - "5432:5432"
     restart: on-failure
     env_file:
-      - ./discord_bot/discord_db/.env
\ No newline at end of file
+      - ./discord_bot/discord_db/.env
+  
+  grafana:
+    image: grafana/grafana:latest
```
Comment (eyalFischel) [inline docker-compose.yml:12]: avoid using "latest" use a specific version.

File: docker-compose.yml [hunk 2]
```diff
@@ -6,4 +6,16 @@ services:
...
+    ports:
+      - "3000:3000"
+    volumes:
+      - grafana_data:/var/lib/grafana
+    env_file:
+      - .env
+
+volumes:
```
Comment (eyalFischel) [inline docker-compose.yml:20]: Again not sure why you are using volumes here.

Comment (Avev) [reply docker-compose.yml:20]: I read its recommended to use them for backup in case the container restarts

Comment (eyalFischel) [reply docker-compose.yml:20]: Ok, so just change the version above and we can merge

Comment (Avev) [reply docker-compose.yml:20]: done

**Pull request closed without merging**